src/org/rascalmpl/interpreter/DefaultTestResultListener.java

```diff
@@ -49,7 +49,7 @@ private void reset() {
     }
 	
 	public void setErrorStream(PrintWriter errorStream) {
-		this.err = errorStream;
+		this.out = errorStream;
```

Comment:
The reason is that I don't know how to wrap the error stream _and_ the output stream at the same time using the current design of the TerminalProgressBarMonitor. Yet it is essential that all output that is printed in the terminal is filtered, otherwise the progress bars end up all over the place in an ugly manner.